kevc/kevc_test.go

```diff
@@ -0,0 +1,74 @@
+package kevc_test
+
+import (
+	"github.com/aquasecurity/vuln-list-update/kevc"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestUpdate(t *testing.T) {
+	tests := []struct {
+		name      string
+		inputFile string
+		wantErr   string
+	}{
+		{
+			name:      "happy path",
+			inputFile: "testdata/happy/known_exploited_vulnerabilities.json",
+		},
+		{
+			name:      "sad path, invalid json",
+			inputFile: "testdata/sad/known_exploited_vulnerabilities.json",
+			wantErr:   "failed to KEVC json unmarshal",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can we use http.FileServer by renaming the file?
https://github.com/aquasecurity/vuln-list-update/blob/87765ef5608bfed13e7c73f5171b55f3017f0c21/go-vulndb/govulndb_test.go#L40

Or http.ServeFile.
https://github.com/aquasecurity/vuln-list-update/blob/3104324321c18b5962ebdf7284206333614d989f/alpine-unfixed/alpine_test.go#L60